content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md

```diff
@@ -54,9 +54,9 @@ Using this phase you can remove this control-plane node from the ClusterStatus o
 {{< /tabs >}}
 
 <!--
-## kubeadm reset 移除 etcd 成员的阶段
+## kubeadm reset phase remove-etcd-member
 -->
-## kubeadm reset phase remove-etcd-member {#cmd-reset-phase-remove-etcd-member}
+## kubeadm reset 移除 etcd 成员的阶段 {#cmd-reset-phase-remove-etcd-member}
```

Comment:
this line should not be translated ...

Reply:
Line 73 translation is incorrect. We are not supposed to translate the command itself.

Reply:
Thanks for your reply! In fact, the changes I made were to be consistent with other heading styles.  As you say, this means that some other command titles should not be translated, these includ `kubeadm init phase`,`kubeadm join phase`,`kubeadm reset phase` and `kubeadm upgradephase`. If you are sure to do this, I can fix all of them.

Reply:
Yes. Please reread the article. The original translation was wrong... they should not be translated.

Reply:
@tengqm I have repushed, and fix all of wrong "kubeadm reset command titles".